LoadFolders.xml

```diff
@@ -388,7 +388,9 @@
 		<li IfModActive="Mlie.CutePenguin">ModPatches/Penguin</li>
 		<li IfModActive="Sov.WarcasketWeapons">ModPatches/Persona Warcasket Weapons</li>
 		<li IfModActive="Vanya.Polarisbloc.CoreLab">ModPatches/Polarisbloc Core Lab</li>
+    <li IfModActive="Vanya.Polarisbloc.CoreLab.tmp">ModPatches/Polarisbloc Core Lab</li>
```

Comment:
`IfModActive` entries targeting the same patch can be combined using a comma--see the entry for the Poleepkwa Race just a couple of lines down.